### PR TITLE
mongo resources takes namespace from release runtime

### PR DESCRIPTION
**Pull request by korenlev**
**Description of the change**

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Comment by Mauraza** [comment]
Hi @korenlev,

Could you explain a little about your change? Why have you changed the examples?
Please check the last item of the checklist.

**Comment by korenlev** [comment]
the purpose is to not use a hard-coded namespace, make it dynamic

failure is environment specific (it works in my build cluster):
"1 node(s) had taints that the pod didn't tolerate."

File: bitnami/mongodb/values.yaml [hunk 1]
```diff
@@ -57,24 +57,24 @@ clusterDomain: cluster.local
...
-##       - name: mongodb
+##       - name: mongodb-primary
 ##         port: 30001
 ##         nodePort: 30001
 ##         protocol: TCP
 ##         targetPort: mongodb
 ##     selector:
 ##       app.kubernetes.io/component: mongodb
-##       app.kubernetes.io/instance: mongodb
+##       app.kubernetes.io/instance: '{{ .Release.Namespace }}'
```
Comment (Mauraza) [inline bitnami/mongodb/values.yaml:77]: app.kubernetes.io/instance is not the '{{ .Release.Namespace }}'

File: bitnami/mongodb/values.yaml [hunk 2]
```diff
@@ -486,10 +486,10 @@ initContainers: {}
 ##   image: korenlev/k8s-mongo-labeler-sidecar
 ##   imagePullPolicy: Always
 ##   env:
-##     - name: LABEL_SELECTOR
-##       value: "app.kubernetes.io/component=mongodb,app.kubernetes.io/instance=mongodb,app.kubernetes.io/name=mongodb"
 ##     - name: NAMESPACE
-##       value: "the-mongodb-namespace"
+##       value: '{{ .Release.Namespace }}'
+##     - name: LABEL_SELECTOR
+##       value: "app.kubernetes.io/component=mongodb,app.kubernetes.io/instance=$(NAMESPACE),app.kubernetes.io/name=mongodb"
```
Comment (Mauraza) [inline bitnami/mongodb/values.yaml:492]: could you move this to the original lines?

**Review by Mauraza — CHANGES_REQUESTED** [review]
Hi @korenlev,

Please check the last item of the checklist: 
- Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

**Pull request closed without merging**